DigitalLearningSolutions.Data/Services/TrackerActionService.cs

```diff
@@ -1,23 +1,39 @@
 ﻿namespace DigitalLearningSolutions.Data.Services
 {
+    using System;
+    using System.Collections.Generic;
     using System.Linq;
     using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Enums;
     using DigitalLearningSolutions.Data.Models.Tracker;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
 
     public interface ITrackerActionService
     {
         TrackerObjectiveArray? GetObjectiveArray(int? customisationId, int? sectionId);
 
         TrackerObjectiveArrayCc? GetObjectiveArrayCc(int? customisationId, int? sectionId, bool? isPostLearning);
+
+        public TrackerEndpointResponse StoreDiagnosticJson(int? progressId, string? diagnosticOutcome);

```

Comment:
`public` access modifier not required here.

Reply:
oops removed :)